packages/frontend/core/src/components/attachment-viewer/index.tsx

```diff
@@ -35,12 +35,12 @@ export const AttachmentViewerView = ({ model }: AttachmentViewerProps) => {
   );
 };
 
-const AttachmentViewerInner = (props: PDFViewerProps) => {
+const AttachmentViewerInner = (props: AttachmentViewerBaseProps) => {
```

Comment:
As we discussed before. This will create a branch in affine that mock the data flow in blocksuite. Which means changes in blocksuite won't be applied to this part. It will case a regression easily. I think we'd better to refactor this part ASAP.